arbitrum/apibackend.go

```diff
@@ -449,3 +502,11 @@ func (a *APIBackend) Engine() consensus.Engine {
 func (b *APIBackend) PendingBlockAndReceipts() (*types.Block, types.Receipts) {
 	return nil, nil
 }
+
+func (b *APIBackend) FallbackClient() types.FallbackClient {
+	if b.fallbackClient == nil && b.fallbackClientUrl != "" {
+		// TODO: reattempt connection?
+		log.Error("fallback client requested, but initial connection failed")
```

Comment:
We should definitely either fail on boot if the connection to the fallback node fails, or reconnect here. As-is, it seems way too likely that we'd have some intermittent networking issue causing connections to the fallback nodes to fail, but the issue would persist long after we fixed the cause because any nitro nodes that started up during that time period would be unable to fallback.